tests/consumers/acero_consumer.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pyarrow as pa
+import pyarrow.substrait as substrait
+
+
+class AceroConsumer:
+    """
+    Implementation of the acero substrait consumer Class for testing
+    """
+
+    @staticmethod
+    def run_substrait_query(substrait_query: str) -> pa.Table:
```

Comment:
```suggestion
    def run_substrait_query(substrait_query: bytes) -> pa.Table:
```
I don't know if you need `bytes` but I'm pretty sure that's what you are receiving here.

Reply:
Taking your suggestion above and doing the encoding within this method.  So now string is what i'm expecting.